src/components/modeler/Modeler.vue

```diff
@@ -1012,7 +1026,21 @@ export default {
         control,
       });
     },
-    async handleDrop({ clientX, clientY, control, nodeThatWillBeReplaced }) {
+    handleDrop(data) {
+      const { clientX, clientY, control} = data;
+      if (this.isMultiplayer) {
+        window.ProcessMaker.EventBus.$emit('multiplayer-addNode', {
+          clientX,
+          clientY,
+          control,
+          id: 'node_' + (this.nodeIdGenerator.getDefinitionNumber()),
```

Comment:
You can use template literals

```suggestion
          id: `node_${this.nodeIdGenerator.getDefinitionNumber()}`,
```